test/integration/expected_out_single_line/escaped_newline.py

```diff
@@ -1,5 +1,6 @@
 from textwrap import dedent
 
+
```

Comment:
Does pre-commit / CI enforce this check? 
if so, it's better removed to help write new tests. flynt should work for python code independent of black formatting, so should also be testable on such.

Reply:
Ah, yeah, I guess `isort` should also be excluded on the integration test victim files. Just a sec.